osu.Game/Tests/Rulesets/TestRulesetConfigCache.cs

```diff
@@ -14,8 +12,8 @@ namespace osu.Game.Tests.Rulesets
     /// </summary>
     public class TestRulesetConfigCache : IRulesetConfigCache
     {
-        private readonly ConcurrentDictionary<string, IRulesetConfigManager> configCache = new ConcurrentDictionary<string, IRulesetConfigManager>();
+        private readonly ConcurrentDictionary<string, IRulesetConfigManager?> configCache = new ConcurrentDictionary<string, IRulesetConfigManager?>();
 
-        public IRulesetConfigManager GetConfigFor(Ruleset ruleset) => configCache.GetOrAdd(ruleset.ShortName, _ => ruleset.CreateConfig(null));
+        public IRulesetConfigManager? GetConfigFor(Ruleset ruleset) => configCache.GetOrAdd(ruleset.ShortName, _ => ruleset.CreateConfig(null));
```

Comment:
I guess `CreateConfig()` should not accept null eventually, right?

Reply:
In test contexts it still should, since the settings are not meant to be stored in a realm database, but only in memory.